addons/binding/org.openhab.binding.russoundserial/README.md

```diff
@@ -0,0 +1,98 @@
+#Russound Serial Binding
+
+This binding allows you to control a [Russound MCA/CAA](https://www.russound.com/products/audio-systems/multi-room-controllers/caa66-system-and-kits/caa66-controller-amplifier) system over a
+serial connection. There is an existing [Russound plugin](https://github.com/openhab/openhab2-addons/tree/master/addons/binding/org.openhab.binding.russound), but that works over a network connection on a different protocol (RIO).
+
+This plugin uses a [kotlin library]([https://github.com/holmes/russound) to communicate via the serial protocol.
+
+## Supported Things
+
+There are 2 supported Thing(s):
+1. Bridge: This is your connection to the controller. You'll specify the serial port once per bridge.
+1. Zones: You'll have a 1:1 mapping of a Zone Thing to a physical zone on your Russound device.
+
+## Discovery and Thing Configuration
+
+There is no auto-discovery. 
+
+#### Bridge
+You must specify the serialPort the Bridge is connected to when setting up the Bridge Thing.
```

Comment:
Sure. I left them the parameter names to give a hint as what you should be setting.